pkg/checker/status.go

```diff
@@ -76,9 +80,21 @@ func (vc StatusControl) setStatus(status string) error {
 		glog.V(4).Infof("failed to update annotation %s for deployment %s in namespace %s: %v", annWebhookStats, deployName, deployNamespace, err)
 		return err
 	}
+	// create event on kyverno deployment
+	createStatusUpdateEvent(status, vc.eventGen)
```

Comment:
This is to create the event on each successful update, is it necessary? Why don't we create event on failure to report unsuccessful action?

Reply:
The event is only created if status changes, not for every update.

Reply:
sorry missed that this is in setStatus